docs/content/latest/api/ysql/syntax_resources/ysql_grammar.ebnf

```diff
@@ -98,8 +149,45 @@ create_index ::= 'CREATE' [ 'UNIQUE' ] 'INDEX' [ [ 'IF' 'NOT' 'EXISTS' ] name ]
 
 index_elem ::= ( column_name | '(' expression ')' ) [ opclass ] [ 'ASC' | 'DESC' ] ;
 
+(* 'CREATE' 'ROLE' *)
+create_role ::= 'CREATE' 'ROLE' role_name [ [ 'WITH' ] role_option { ',' role_option } ] ;
+
+role_option ::= 'SUPERUSER'
+              | 'NOSUPERUSER'
+              | 'CREATEDB'
+              | 'NOCREATEDB'
+              | 'CREATEROLE'
+              | 'NOCREATEROLE'
+              | 'INHERIT'
+              | 'NOINHERIT'
+              | 'LOGIN'
+              | 'NOLOGIN'
+              | 'CONNECTION' 'LIMIT' connlimit
+              | [ 'ENCRYPTED' ] 'PASSWORD' ''' password '''
+              | 'PASSWORD' 'NULL'
+              | 'VALID' 'UNTIL' ''' timestamp '''
+              | 'IN' 'ROLE' role_name { ',' role_name }
+              | 'IN' 'GROUP' role_name { ',' role_name }
+              | 'ROLE' role_name { ',' role_name }
+              | 'ADMIN' role_name { ',' role_name }
+              | 'USER' role_name { ',' role_name }
+              | 'SYSID' uid ;
+
+role_name ::= '<Text Literal>';
+password ::= '<Text Literal>' ;
+timestamp ::= '<DateTime Literal>' ;
+connlimit ::= '<Integer Literal>';
+uid ::= '<Text Literal>';
+
 (* 'CREATE' 'SCHEMA' *)
-create_schema ::= 'CREATE' 'SCHEMA' [ 'IF' 'NOT' 'EXISTS' ] schema_name [ schema_element { schema_element } ] ;
+create_schema_name ::= 'CREATE' 'SCHEMA' [ 'IF' 'NOT' 'EXISTS' ] schema_name
+                       [ 'AUTHORIZATION' role_specification ]
+                       [ schema_element { schema_element } ] ;
+
+create_schema_role ::= 'CREATE' 'SCHEMA' [ 'IF' 'NOT' 'EXISTS']
+                       'AUTHORIZATION' role_specification [ schema_element { schema_element } ] ;
+
+schema_element ::= '<YSQL DDL statement>' ;
```

Comment:
Can you put a TODO for me here as a comment `(* TODO add ddl_statement, dcl_statement, dml_statement as top level rules  *)`.